corehq/apps/ota/views.py

```diff
@@ -104,6 +104,7 @@ def app_aware_search(request, domain, app_id):
     """
     criteria = request.GET.dict()
     try:
+        # could be a list or a single string
```

Comment:
Is this still true? It looked like you updated this to a list in a couple of places not directly to this PR, which made me think you'd switched to a list everywhere.

And if it **is** still true, would it be arduous to switch it to `case_types` and make it always a list?

Reply:
Yes. Any param can be a list, since we added the multiselect support. 

> would it be arduous to switch it to case_types and make it always a list?

It would require changes to the suite xml structure which always had a `case_type`. So, it's probably better to keep it as it is.